app/modules/frontend/resources.py

```diff
@@ -56,7 +54,9 @@ def parse_frontend_versions():
             frontend_datetime = datetime.datetime.strptime(
                 frontend_timestamp, DATETIME_FMTSTR
             )
-            frontend_datetime = frontend_datetime.replace(tzinfo=PST)
+            frontend_datetime = frontend_datetime.replace(
+                tzinfo=current_app.config.get('TIMEZONE')
+            )
             frontend_time = frontend_datetime.strftime('%B %d, %Y, %H:%M:%S %p Pacific')
```

Comment:
This line is now wrong, that should no longer say Pacific but I don't know what this code is doing or how to test it
My thought was that this string also needs to be some sort of configuration value but no idea what it should be called